src/Blast/Component/UI/Model/CustomReportOwnerInterface.php

```diff
@@ -0,0 +1,25 @@
+<?php
```

Comment:
Maybe add a CustomReportOwnerTrait ?

Reply:
I haven't done it because there is no specific logic to apply to the inverse side of a Doctrine bidirectional relation.